benchmarks/src/jmh/kotlin/benchmarks/flow/FlowFlattenMergeBenchmark.kt

```diff
@@ -0,0 +1,52 @@
+package benchmarks.flow
+
+import doWork
+import kotlinx.coroutines.flow.*
+import kotlinx.coroutines.runBlocking
+import org.openjdk.jmh.annotations.*
+import java.util.concurrent.TimeUnit
+
+@Warmup(iterations = 3)
```

Comment:
Please, specify the batch size as ELEMENTS, see the link below
https://github.com/Valloric/jmh-playground/blob/master/src/jmh/java/org/openjdk/jmh/samples/JMHSample_26_BatchSize.java

Reply:
If I change annotations like that the benchmark doesn't compute anything